pkg/distro/packagesets/loader.go

```diff
@@ -0,0 +1,95 @@
+package packagesets
+
+import (
+	"embed"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/distro"
+	"github.com/osbuild/images/pkg/rpmmd"
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed */*.yaml
+var Data embed.FS
+
+type packageSet struct {
+	Include   []string   `yaml:"include"`
+	Exclude   []string   `yaml:"exclude"`
+	Condition conditions `yaml:"condition,omitempty"`
```

Comment:
Shouldn't this be a pointer? I think it's quite common to have optional properties as pointers.

Reply:
Done!